### ISSUE TITLE
Remove @vue/composition-api from peerDependencies

### ISSUE THREAD

**Issue by IlyaSemenov** [TownSquareXYZ/tonconnect-ui-vue#17]
`@townsquarelabs/ui-vue` lists `@vue/composition-api` as a peer dependency:

https://github.com/TownSquareXYZ/tonconnect-ui-vue/blob/ee614c6b09efb3a9ef200887882a88de32a300d6/package.json#L61

There is no point in doing so. The package never imports or uses `@vue/composition-api` in any other way. As such, `@vue/composition-api` should not be listed as a dependency.

Having it pulls unnecessary libraries in Vue 3 projects and pollutes output with warnings:

```
 WARN  Issues with peer dependencies found
.
└─┬ @vue/composition-api 1.7.2
  └── ✕ unmet peer vue@">= 2.5 < 2.7": found 3.4.37
```

**Comment by kkkkh** [comment]
> `@townsquarelabs/ui-vue` lists `@vue/composition-api` as a peer dependency:
> 
> https://github.com/TownSquareXYZ/tonconnect-ui-vue/blob/ee614c6b09efb3a9ef200887882a88de32a300d6/package.json#L61
> 
> There is no point in doing so. The package never imports or uses `@vue/composition-api` in any other way. As such, `@vue/composition-api` should not be listed as a dependency.
> 
> Having it pulls unnecessary libraries in Vue 3 projects and pollutes output with warnings:
> 
> ```
>  WARN  Issues with peer dependencies found
> .
> └─┬ @vue/composition-api 1.7.2
>   └── ✕ unmet peer vue@">= 2.5 < 2.7": found 3.4.37
> ```

@IlyaSemenov 
Yes, you are right. The package after building does not need "peerDependencies": {
"@vue/composition-api": "^1.0.0-rc.1",}. The reason why it exists in package.json is because this dependency is needed during development. Later, we will separate the build and dev of the code. The package after building will remove unnecessary dependencies.

**Comment by supakornkim** [comment]
is this fix already?

or someone please tell me how to solve this?

**Comment by IlyaSemenov** [comment]
@supakornkim I use `.pnpmfile.cjs` to handle this:

```js
function readPackage(pkg) {
  // See https://github.com/TownSquareXYZ/tonconnect-ui-vue/issues/17
  if (pkg.name === "@townsquarelabs/ui-vue" && pkg.version === "2.0.11-beta.0") {
    delete pkg.peerDependencies["@vue/composition-api"]
  }
  return pkg
}

module.exports = {
  hooks: {
    readPackage,
  },
}
```
